android/sdl_android/src/main/java/com/smartdevicelink/managers/lockscreen/SDLLockScreenActivity.java

```diff
@@ -64,6 +69,9 @@
 	private static final int MIN_SWIPE_DISTANCE = 200;
 	private boolean mIsDismissible;
 	private GestureDetector mGestureDetector;
+	private int backgroundColor = Color.parseColor("#394e60");
```

Comment:
Should "#394e60" be some sort of magic number? or maybe we can name backgroundColor something like defaultBackgroundColor

Reply:
That is the default background color, I will change the name.

Reply:
Agreed, I was missing some of the context